data/features/css-file-selector-button.js

```diff
@@ -0,0 +1 @@
+module.exports={A:{D:{"1":"0 1 2 3 4 5 6 7 8 9 lB cB I e J E F G A B C K L D M N O f g h i j k l m n o p q r s t u v w x y z AB BB CB DB EB FB GB HB IB dB JB eB KB LB S MB NB OB PB QB RB SB TB UB VB WB XB YB ZB P Q T U V W X Y Z a b c R d H fB pB"},L:{"1":"0 1 2 3 4 5 6 7 8 9 O l m n o p q r s t u v w x y z AB BB CB DB EB FB GB HB IB dB JB eB KB LB S MB NB OB PB QB RB SB TB UB VB WB XB YB ZB P Q T U V W X Y Z a b c R d H fB pB"},B:{"1":"C K L D M N O P Q T U V W X Y Z a b c R d H fB pB"},C:{"1":"mB U V W X Y Z a b c R d H fB pB qB undefined","2":"0 1 2 3 4 5 6 7 8 9 lB cB I e J E F G A B C K L D M N O f g h i j k l m n o p q r s t u v w x y z AB BB CB DB EB FB GB HB IB dB JB eB KB LB S MB NB OB PB QB RB SB TB UB VB WB XB YB ZB P Q T nB oB"},M:{"1":"mB U V W X Y Z a b c R d H fB pB qB undefined","2":"0 1 2 3 4 5 6 7 8 9 I e J E F G A L D M N O f g h i j k l m n o p q r s t u v w x y z AB BB CB DB EB FB GB HB IB dB JB eB KB LB S MB NB OB PB P Q T"},A:{"1":"A B","2":"lB cB I e J E F G kB"},F:{"1":"0 1 2 3 4 5 6 7 8 9 D M N O f g h i j k l m n o p q r s t u v w x y z AB BB CB DB EB FB GB HB IB JB KB LB S MB NB OB PB QB RB SB TB UB VB WB XB YB ZB P Q T","2":"lB cB I e J E F G A B C nB oB sB uB vB ZC hB 1B 2B aB iB 3B bB"},K:{"1":"1 2 3 4 5 6 7 8 9 L D M O f g h i k l m n o p q s t u v w x AB BB CB DB EB FB GB HB IB dB JB eB KB LB S","2":"B C hB aB iB bB"},E:{"1":"cB I e J E F G A B C K L D rB sB tB vB hB aB bB wB xB undefined","2":"lB"},G:{"1":"lB cB I e J E F G A B C K L D gB tB AC CC"},P:{"1":"OC gB YC ZC bC bB cC IC dC eC"},I:{"1":"0 1 2 3 4 5 6 7 8 9 lB cB I x y z AB BB CB DB EB FB GB HB IB dB JB eB KB LB S MB NB OB PB QB RB SB TB UB VB WB XB YB ZB P Q T U V W X Y Z a b c R d H fB pB PC SC"}},B:6,C:"::file-selector-button CSS pseudo-element"};
```

Comment:
Some `undefined` values in here, could you try updating this branch against the latest caniuse-db to see if it fixes these please? :slightly_smiling_face:

Reply:
Taking a look at this now. Also noticed an issue where the MDN data is being treated as full support when in fact it should be prefixed.

Reply:
This should now be dealt with